pkg/controller/replication_controller.go

```diff
@@ -36,10 +36,6 @@ import (
 	"github.com/golang/glog"
 )
 
-var (
-	rcKeyFunc = framework.DeletionHandlingMetaNamespaceKeyFunc
```

Comment:
Everything to do with rcKeyFunc will have merge conflicts anyway, because I generalized Expectations :)